public/app/features/dashboard/components/ShareModal/SharePublicDashboard/Configuration.tsx

```diff
@@ -36,6 +40,19 @@ export const Configuration = ({
             <Label description="The public dashboard uses the default time settings of the dashboard">Time range</Label>
             <TimeRangeInput value={timeRange} disabled onChange={() => {}} />
           </Layout>
+          <Layout orientation={isDesktop ? 0 : 1} spacing="xs" justify="space-between">
+            <Label description="Show annotations on public dashboard">Show Annotations</Label>
```

Comment:
We should use sentence case

```suggestion
            <Label description="Show annotations on public dashboard">Show annotations</Label>
```